--- a/third_party/ffmpeg/ffmpeg_generated.gni
+++ b/third_party/ffmpeg/ffmpeg_generated.gni
@@ -315,5 +315,9 @@
   ffmpeg_c_sources += [
+    "libavcodec/aom_film_grain.c",
     "libavcodec/bswapdsp.c",
+    "libavcodec/dovi_rpu.c",
+    "libavcodec/dovi_rpudec.c",
+    "libavcodec/dynamic_hdr_vivid.c",
     "libavcodec/hevc/autorename_libavcodec_hevc_parse.c",
     "libavcodec/hevc/autorename_libavcodec_hevc_parser.c",
     "libavcodec/hevc/autorename_libavcodec_hevc_cabac.c",
